apps/designer/.storybook/preview.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import { TooltipProvider } from "@radix-ui/react-tooltip";
 import { globalCss } from "@webstudio-is/design-system";
+import { variables } from "@webstudio-is/design-system";
```

Comment:
variables in js context is a bit unclear, what about themeVars?  or stitchesVars? since its a particular format for the value defined by stitches?

Reply:
Talked in dm. Decided to use `theme`. But need to rename an existing `theme` export in `stitches.config.ts`. Will rename in separate PR.